### ISSUE TITLE
Sniper triggered a BUY order even if no liquidity was added

### ISSUE THREAD

**Issue by tsarbuig** [CryptoGnome/Limit-Sniper#32]
a user reported me that his Sniper bot tried to buy this token, even if no liquidity was added, as you can see with the timestamp...

Do you have any idea of which event could have triggered a BUY order ?

- token : https://bscscan.com/address/0x418236ca7c807cfc1d04eb64475df7ce17c2f218

- tokens.json : 
[
  {
    "ADDRESS": "0x418236CA7c807Cfc1D04eB64475df7cE17C2F218",
    "SYMBOL": "DuxDoge",

    "USECUSTOMBASEPAIR": "false",
    "BASESYMBOL": "USDT",
    "BASEADDRESS": "0xdac17f958d2ee523a2206206994597c13d831ec7",

    "BUYAMOUNT": "0.01",
    "MULTIPLEBUYS": "false",
    "BUYCOUNT": "1",
    "RUGCHECK": "false"
  }
]

- failed transaction : https://bscscan.com/tx/0xdb7ae12dad5440945c4d67bcecdfebafb8c27496290c5c9128410d83bafca63a

**Comment by CryptoGnome** [comment]
is this dx sale?

**Comment by alefantozzi** [comment]
> is this dx sale?

No, it was a token trading on PCS for s very long time. I had just sold tokens using the bot without a problem, but when trying to buy again got this issue.
